mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java

```diff
@@ -35,6 +35,7 @@
   public static final Logger LOGGER = LoggerFactory.getLogger(JobExecutionSourceChunkDaoImpl.class);
   private static final String TABLE_NAME = "job_execution_source_chunks";
   private static final String ID_FIELD = "'id'";
+  private static final String GET_PROCESSING_IS_COMPLETED_AND_HAS_ERROR_QUERY = "SELECT get_processing_is_completed_and_has_error('%s');";
```

Comment:
I would name it as GET_PROCESSING_STATUS. The "get" and "is" and "has" are a bit confusing me especially in one sentences

Reply:
In my opinion, it's an overengineering. There are no cases to do this solution badly. So I propose to close this one